module-2/lab-subsetting-and-descriptive-stats/your-code/main.ipynb

```diff
@@ -1412,7 +1412,7 @@
     {
```

Comment:
- It's a good practice to skip useless columns when importing data. This is the case of the first column of the orders csv file, It avoids misunderstanding. You can do this staff using usecols pandas within read_csv method. 
- I don't see the comments with the interpretation of the histograms and the boxplots.